documentation/modules/ROOT/partials/modules/tutorial/con-introduction-debezium.adoc

```diff
@@ -1,19 +1,30 @@
-// Metadata created by nebel
-//
-// UserStory:
-
 [id="introduction-debezium"]
 = Introduction to {prodname}
 
-{prodname} is a distributed platform that turns your existing databases into event streams,
-so applications can see and respond immediately to each row-level change in the databases.
+{prodname} is a distributed platform that converts information from your existing databases into event streams, enabling applications to detect, and immediately respond to row-level changes in the databases.
+
+{prodname} is built on top of http://kafka.apache.org[Apache Kafka] and provides a set of {link-kafka-docs}.html#connect[Kafka Connect] compatible connectors.
+Each of the connectors works with a specific database management system (DBMS).
+Connectors record the history of data changes in the DBMS by detecting changes as they occur, and streaming a record of each change event to the Kafka logs.
+Consuming applications can then read the resulting event records from the log.
```

Comment:
I wonder if using the term `topics` rather than `logs` would be more appropriate in this context.

Reply:
Ha, funny that you should say that. I had changed it, but reverted it to `logs`, because I thought that maybe the original writer intentionally used that language for some reason. I'm very happy to go with `topics`!

Reply:
I think from a technical perspective someone really close to Kafka likely might refer to it as `logs`, but when you move just beyond that to the common lay person, the `topics` term seems more appropriate and clearer to me.  It also avoids any confusion in context as we often refer to "transaction logs" from a CDC perspective and this keeps that distinction crystal clear to me.